config.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget id="uk.gov.dvsa.mobile-examiner" version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+<widget id="uk.gov.dvsa.mobile-examiner" version="1.0.1543422786" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
```

Comment:
Version bumped.

Reply:
Perhaps we should look into only running the versioning hook when a flag is provided to the build command? (Providing this is something we could configure on Ionic Pro.

Reply:
Thanks for spotting, I was meant to revert this back to what's on develop, will do it now

Reply:
And yes, should only bump the version when we build on Ionic Pro

Reply:
Okay, will have to look into what we can configure for this on Ionic Pro and update the build hook accordingly.